### PR TITLE
Indent vtk related files

### PR DESCRIPTION
**Pull request by Davknapp**
Added all vtkDatatypes to the custom datatypes file to improve indentation

**_All these boxes must be checked by the reviewers before merging the pull request:_**

As a reviewer please read through all the code lines and make sure that the code is fully understood, bug free, well-documented and well-structured.

#### General
- [ ] The reviewer executed the new code features at least once and checked the results manually

- [ ] The code follows the [t8code coding guidelines](https://github.com/holke/t8code/wiki/Coding-Guideline)
- [ ] New source/header files are properly added to the Makefiles
- [ ] The code is well documented
- [ ] All function declarations, structs/classes and their members have a proper doxygen documentation
- [ ] All new algorithms and data structures are sufficiently optimal in terms of memory and runtime (If this should be merged, but there is still potential for optimization, create a new issue)

#### Tests
- [ ] The code is covered in an existing or new test case using Google Test

#### Github action

- [ ] The code compiles without warning in debugging and release mode, with and without MPI (this should be executed automatically in a github action)
- [ ] All tests pass (in various configurations, this should be executed automatically in a github action)

  If the Pull request introduces code that is not covered by the github action (for example coupling with a new library):
  - [ ] Should this use case be added to the github action?
  - [ ] If not, does the specific use case compile and all tests pass (check manually)

#### Scripts and Wiki

- [ ] If a new directory with source-files is added, it must be covered by the `script/find_all_source_files.scp` to check the indentation of these files.
- [ ] New Datatypes are added to t8indent_custom_datatypes.txt
- [ ] If this PR introduces a new feature, it must be covered in an example/tutorial and a Wiki article.

#### Licence

- [ ] The author added a BSD statement to `doc/` (or already has one)

### CODE REVIEW

File: src/t8_forest/t8_forest_vtk.cxx [hunk 1]
```diff
@@ -237,6 +237,7 @@ t8_forest_vtk_get_element_nodes (t8_forest_t forest, t8_locidx_t ltreeid,
  * the data related to the element (not element_data). 
  * 
  */
+/* *INDENT-OFF* */
```
Comment (sandro-elsweijer) [inline src/t8_forest/t8_forest_vtk.cxx:240]: Do we need this indent off with the newly added datatypes?

File: src/t8_vtk/t8_vtk_parallel.hxx [hunk 2]
```diff
@@ -56,8 +56,8 @@ vtk_read_success_t  t8_read_parallel_unstructured (const char *filename,
  * \returns        non-zero on success, zero if the reading failed. 
  */
 vtk_read_success_t  t8_read_parallel_polyData (const char *filename,
-                                               vtkSmartPointer < vtkDataSet >
-                                               grid, sc_MPI_Comm comm);
+                                               vtkSmartPointer < vtkDataSet
+                                               >grid, sc_MPI_Comm comm);
```
Comment (sandro-elsweijer) [inline src/t8_vtk/t8_vtk_parallel.hxx:60]:
```suggestion
                                               vtkSmartPointer < vtkDataSet >
                                               grid, sc_MPI_Comm comm);
```

File: src/t8_vtk/t8_vtk_reader.hxx [hunk 3]
```diff
@@ -48,8 +48,9 @@ T8_EXTERN_C_BEGIN ();
  * \param[in] comm The communicator. 
  * \return t8_cmesh_t 
  */
-t8_cmesh_t          t8_vtkGrid_to_cmesh (vtkSmartPointer < vtkDataSet >
-                                         vtkGrid, const int partition,
+t8_cmesh_t          t8_vtkGrid_to_cmesh (vtkSmartPointer <
+                                         vtkDataSet >vtkGrid,
+                                         const int partition,
```
Comment (sandro-elsweijer) [inline src/t8_vtk/t8_vtk_reader.hxx:53]:
```suggestion
t8_cmesh_t          t8_vtkGrid_to_cmesh (vtkSmartPointer
                                         < vtkDataSet >vtkGrid,
                                         const int partition,
```

**Review by sandro-elsweijer — CHANGES_REQUESTED** [review]
It still looks ugly, but at least it is now the fault of gnu indent

File: src/t8_forest/t8_forest_vtk.cxx (hunk 1 above)
Comment (Davknapp) [reply src/t8_forest/t8_forest_vtk.cxx:240]: Yes, otherwise gnuindent changes it to an ugly mess.

Comment (sandro-elsweijer) [reply src/t8_forest/t8_forest_vtk.cxx:240]: kk

**Pull request merged**